crates/runtime/src/databackend/sqlite.rs

```diff
@@ -75,7 +78,20 @@ impl DataPublisher for SqliteBackend {
     fn add_data(&self, _dataset: Arc<Dataset>, data_update: DataUpdate) -> AddDataResult {
         let pool = Arc::clone(&self.pool);
         let name = self.name.clone();
+
         Box::pin(async move {
+            if let Some(batch) = data_update.data.first() {
+                for field in batch.schema().fields() {
+                    if field.data_type().is_nested() {
+                        let field_name = self.name.clone() + "." + field.name();
```

Comment:
use `format!()` to create the field_name

Reply:
Updated